linux-deploy.sh

```diff
@@ -1,21 +1,15 @@
-# node should be installed
+# node and npm should be installed
 
 if (which node)
 then 
- echo "node is installed"
+  echo "node is installed"
 else
- echo "installing node"
- apt install node
+  echo "installing node"
+curl -SLO https://deb.nodesource.com/nsolid_setup_deb.sh 
+chmod 500 nsolid_setup_deb.sh
```

Comment:
This was the issue, you changed this file to read only. Not a good idea.

Reply:
I copied and pasted, but what makes it read only?

Reply:
the `chmod 500` is setting the file permissions to `r-x` because it's `101 000 000`.